snapcraft/tests/unit/test_lxd.py

```diff
@@ -537,6 +537,49 @@ def make_containerbuild(self):
                            project_options=self.project_options,
                            remote=self.remote)
 
+    @patch('snapcraft.internal.lxd.Containerbuild._container_run')
+    @patch('snapcraft.internal.common.is_snap')
+    def test_inject_snap_existing_container(self,
```

Comment:
What makes this container already-existing?

Reply:
Apparently I didn't commit that bit... fixed